components/crud-web-apps/jupyter/frontend/src/index.html

```diff
@@ -6,10 +6,6 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
-    <link
```

Comment:
Could you also remove this import from the volumes and tensorboards web apps as well?